saleor/dashboard/discount/forms.py

```diff
@@ -43,6 +59,29 @@ class VoucherForm(forms.ModelForm):
     class Meta:
         model = Voucher
         exclude = ['limit', 'apply_to', 'product', 'category']
+        labels = {
+            'type': pgettext_lazy(
+                'Voucher form label',
+                'Discount type'),
+            'name': pgettext_lazy(
+                'Voucher form label',
+                'Name'),
+            'code': pgettext_lazy(
+                'Voucher form label',
+                'Code'),
+            'usage_limit': pgettext_lazy(
+                'Voucher form label',
+                'Limit for using the voucher'),
+            'used': pgettext_lazy(
+                'Voucher form label',
+                'Number of voucher usages'),
+            'start_date': pgettext_lazy(
+                'Voucher form label',
+                'Date voucher starts being valid'),
+            'end_date': pgettext_lazy(
+                'Voucher form label',
+                'Date voucher stops being valid'),
+        }
```

Comment:
Without hanging brackets the code will look much better. Please correct this one and other hanging brackets that this PR introduces.